format match and index pages

Mathboy19 opened the pull request:
This commit adjusts the formatting for the match page to be a
little cleaner and easier to read. It also changes the index and
footer, adding in some margins for appearance.

![example image](https://cdn.discordapp.com/attachments/488110864780754944/535960887379361812/unknown.png)

flubbateios commented:
Hi. Thanks for contributing to Super Stats. 

The 90% window width change does indeed improve readability and is definitely a good change.
I also really like the table change for the players list although the grey should be slightly darker to be more in line with the rest of the page I think.

I'm not sure why you switched to build bar icons as the default over strategic icons ( I myself prefer strategic icons ). The SIs are in need of a fix for orbital units and that will be coming fairly soon (there's some weird artifacts on the borders). 

The margin-top change for the recent games page is good but also needs to be added to `advancedsearch.html` but is otherwise a good change to put the page number more in line with the buttons. 

The footer improvement is nice but display:table should in general be avoided for things that aren't tables.

Mathboy19 commented:
Thanks for your feedback! I adjusted the colors on the table to be a little duller.
![image](https://user-images.githubusercontent.com/5177788/51435987-743f3300-1c49-11e9-97b8-eb726a3acc35.png)

I switched the icons because I think that the Build Bar Icons are easier to read, especially if you're a newer player to the game who hasn't memorized all of the SI icons yet. Although it _is_ entirely personal preference. Another alternative is to do some combination of the two, like this: 
![vehicle_factory_combined](https://user-images.githubusercontent.com/5177788/51436035-af8e3180-1c4a-11e9-9bde-77e1f7eeab09.png)

I added the margin fix to `advancedsearch.html`, although it should probably be put in a css file. Unfortunately I didn't have the opportunity to test that page because I don't have the backend setup. Is there anyway to make a local server for testing purposes?

flubbateios commented:
Alright. Merging this and I'll merge into release once I get the SI fix done.
